opentracing_shim/README.md

```diff
@@ -1,6 +1,20 @@
+[![Javadocs][javadoc-image]][javadoc-url]
 # OpenTelemetry - OpenTracing Shim
+A bridge layer from OpenTelemetry to the OpenTracing API.
+It takes OpenTelemetry Tracer and expose it as an OpenTracing Tracer.
+
+## Usage
+
+1. Configure OpenTelemetry tracer
```

Comment:
Can we mention we are talking about the global instance?

Reply:
done